regression_detector.py

```diff
@@ -412,7 +412,7 @@ def get_metrics_by_date(
     if not start_date:
         raise RuntimeError(
             f"No start date in previous JSONS found to compare towards the end date {end_date}. User specified start date: {args.start_date}. "
-            + f"Available JSON dates: {available_metrics_jsons.keys()}. No regression info has been generated."
```

Comment:
available_metrics_jsons doesn't have keys anymore as it's a List[str] and not a dict. It's unclear whether there were ever keys. Just remove the printout for now as it's not needed for the error message.